### ISSUE TITLE
Problem with cloning on Windows

### ISSUE THREAD

**Issue by King2500** [aik099/PhpStormProtocol#9]
I can't clone the repository on Windows without problems. :-(

There is an Carriage Return character (%0D) at the end of the `Icon` file name:
https://github.com/aik099/PhpStormProtocol/blob/master/PhpStorm%20Protocol.app/Icon%0D <--

Is that intended? 
Windows has problems with such "special characters" in filenames.
GitHub doesn't display it.

**Comment by King2500** [comment]
Is the `Icon` file needed at all? It has no content.

**Comment by aik099** [comment]
It has content, when viewed on Mac. It's not needed for Windows.

**Comment by aik099** [comment]
Can't download a tarball instead (zipped git clone)?

**Comment by King2500** [comment]
No. Need to clone to update/rebase my fork.

**Comment by aik099** [comment]
I have no idea how this can be fixed. Maybe Google knows. Maybe you can search for that error you're getting during clone. I don't think Git doesn't like empty files in general, but maybe it has having problems detecting file type or something else in that file.

**Comment by King2500** [comment]
```
>git clone https://github.com/King2500/PhpStormProtocol.git
Cloning into 'PhpStormProtocol'...
remote: Counting objects: 85, done.
remote: Compressing objects: 100% (5/5), done.
remote: Total 85 (delta 0), reused 0 (delta 0), pack-reused 80
Unpacking objects: 100% (85/85), done.
 (Invalid argument)eate file PhpStorm Protocol.app/Icon
```
Git isn't able to save the file on Windows systems, since such a character is not allowed on Windows filesystem.
Internet says delete the Icon file or use a Mac (which I can't).

I went another way, by sending a pull-request to my fork and merge the changes from your repos.

In the end I want to make this pull-request, which now includes an additional "merge" commit:
https://github.com/aik099/PhpStormProtocol/compare/master...King2500:King2500-patch-1?expand=1

Is that a problem for you?

**Comment by aik099** [comment]
The file was named `Icon\r` and I've tested that after it was removed Mac app was working too and icon was showing. I've made a commit. Now you shouldn't have any issues with cloning/rebasing/etc. Please send appropriate PR and I'll review it.

**Comment by King2500** [comment]
Got fixed with commit cc2fd3b95b55df4a6fc4ac7736ac55b61b195a9b
